client/src/interface/User.ts

```diff
@@ -1,7 +1,11 @@
 export interface User {
+  // TODO: needs to replace _id with id when server sends response. (I will fix it when integrating)
```

Comment:
You can remove the part where _(I will fix it when integrating)_

Reply:
Applied